src/configHandler.ts

```diff
@@ -66,4 +67,36 @@ export default class ConfigHandler {
 
         return resources;
     }
+
+    getResourceHandler(resourceType: string): ResourceHandler | undefined {

```

Comment:
I think a small comment explaining the method will help

```suggestion
    // @returns specialized resourceHandler, if not found returns generic resourceHandler, if not valid returns undefined
    getResourceHandler(resourceType: string): ResourceHandler | undefined {

```

Reply:
added comment